examples/brandai/README.md

```diff
@@ -0,0 +1,46 @@
+# React-WinJS Component Documentation (see it live [here](https://brandai.com/styleguide/WinJS))
+
+This folder contains configuration required to build documentation for react-winjs using [Brand AI](https://brand.ai).
+
+## Flow
+### Build
+Use gulp to build a distribution folder containing:
+* library version of react-winjs
+* WinJS core scripts and stylesheets.
+
+See [webpack.config.json](https://github.com/winjs/react-winjs/blob/master/examples/brandai/webpack.config.js) and [gulpfile.json](https://github.com/winjs/react-winjs/blob/master/examples/brandai/gulpfile.js) for more info on that.
+
+```sh
+$ npm install
+$ gulp build
+```
+
+### Deploy
+Deploying to Brand.ai is done through [brandai-tools](https://github.com/brandai/brandai-tools).
+```sh
+$ npm install -g brandai-tools
+$ npm login
+$ npm deploy
+```
+
+### Config
+[brandai.json](https://github.com/brandai/react-winjs/blob/master/examples/brandai/brandai.json) contains all the component extraction and deployment information.
+
+* `name` - the component library name in Brand.ai - each documentation project is associated with one or more component libraries
+* `version` - the component library version (in case it is different from the package.json version)
+* `dist` - name of the dist folder containing the library and its dependencies - used both for rendering examples and for dynamic analysis of components and their properties
+* `libraryObjectName` - name of the library on the global object (we set it to `ReactWinJS` in `webpack.config.js`.
+* `reactVersion` - the version of React required for running this library (automatically gets pulled when rendering the components)
+* `dynamicExtraction` - this tells Brand.ai to dynamically extract components - run the files in the `dist` folder and extract the required info from live components, rather than statically parse .jsx files.
+* `fileOrder` - when loading dependencies, file names appearing here will be loaded first (we can't load react-winjs until the winjs core files are there).
+
+```json
```

Comment:
Is this just an example of a config file? If so, can you note that with some text above it.

For example, "Sample config file:" followed by the JSON for the sample config file.